manifests/addons/gen.sh

```diff
@@ -81,3 +83,16 @@ function compressDashboard() {
     --from-file=istio-mesh-dashboard.json="${TMP}/istio-mesh-dashboard.json" \
     --from-file=istio-extension-dashboard.json="${TMP}/istio-extension-dashboard.json"
 } > "${ADDONS}/grafana.yaml"
+
+# Set up loki
+{
+  helm3 template loki loki-stack \
+    --namespace istio-system \
+    --version "${LOKI_VERSION}" \
+    --repo https://grafana.github.io/helm-charts \
+    -f "${WD}/values-loki.yaml"
+} > "${ADDONS}/loki.yaml"
+
+# yq can not output to same file
+yq -I2 "${ADDONS}/loki.yaml" > "${ADDONS}/loki-format.yaml"
```

Comment:
can just pipe from helm to yq

Reply:
but also not clear why we need to?

Reply:
We don't need to lint external things, can we exclude it from the lint

Reply:
or fix upstream chart

Reply:
I will remove this after fix it from upstream(I think this's the right way).

Reply:
@howardjohn `loki` has a major changes form 3.x, which is more complex. I cannot fix it from upstream.

Alternatives, we can move to use loki 3.x, WDYT?

Reply:
Given this is new usage I think we should probably use the latest version? unless there is some reason not to?

Reply:
the new arch in 3.x is a little complex for test-only.

Reply:
updated with latest loki chart.